src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -313,6 +343,46 @@ export const VolunteerFormCalendar = ({
           control={control}
         />
       </Wrapper>
+      <Wrapper>
+        <Controller
+          name="images"
+          render={(field) => (
+            <ImageSelector
+              {...{
+                control,
+                field,
+                isVolunteer: true,
+                item: {
+                  name: 'images',
+                  label: texts.volunteer.images,
+                  rules: { required: false },
+                  buttonTitle: texts.volunteer.addImage
+                }
+              }}
+            />
+          )}
+          control={control}
+        />
+        <Controller
+          name="documents"
+          render={(field) => (
+            <DocumentSelector
+              {...{
+                control,
+                field,
+                isVolunteer: true,
+                item: {
+                  name: 'documents',
+                  label: texts.volunteer.documents,
+                  rules: { required: false },
```

Comment:
is `rules: { required: false },` needed in case it is not required?

Reply:
yes, that's not necessary.
https://github.com/ikuseiGmbH/smart-village-app-app/pull/606/commits/a639aac9df2185059b0baa42df9268df5d95b3db